src/main/java/com/wolox/training/models/User.java

```diff
@@ -0,0 +1,106 @@
+package com.wolox.training.models;
+
+import com.wolox.training.exceptions.BookAlreadyOwnedException;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Past;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(unique = true)
+    @NotEmpty
+    private String username;
```

Comment:
The username and name are fields that should be required or at least username.

What I would do is add in both the nullable false in the @Column of each field and I would also change the @NotEmpty for @NotBlank. The latter not only checks that it is not empty but also checks that it is not null.

```suggestion
    @NotBlank
    @Column(nullable = false, unique = true)
    private String username;
```